doc/moveit_cpp/src/moveit_cpp_tutorial.cpp

```diff
@@ -6,7 +6,7 @@
 
 #include <geometry_msgs/msg/point_stamped.h>
 
-/* #include <moveit_visual_tools/moveit_visual_tools.h>  This has not been ported to ros2 yet */
+#include <moveit_visual_tools/moveit_visual_tools.h>
 #include <rviz_visual_tools/rviz_visual_tools.hpp>
```

Comment:
I think you can remove rviz visual tools include here as mvt already inherits it.